### PR TITLE
fix: nvidia-device-plugin + manual steps taken for testing

### PR DESCRIPTION
**Pull request by rawlingsj**
Thanks to @k4leung4 for the manual testing steps.

### CODE REVIEW

File: images/nvidia-device-plugin/tests/README.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+# Notes on how this image was tested
+
+The `nvidia-device-plugin` image required GPUs to test, therefore it is not currently covered by automated tests.
+
+This document describes the manual steps taken to test the image on GCP.
+
+Note, for testing we use a the `nvidia/gpu-operator` helm chart which requires bash so you will want to use the `latest-dev` variant.
+
+```
+cgr.dev/chainguard-private/nvidia-device-plugin:latest-helm
```
Comment (k4leung4) [inline images/nvidia-device-plugin/tests/README.md:10]: should this be `latest-dev`

File: images/nvidia-device-plugin/tests/README.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
+# Notes on how this image was tested
+
+The `nvidia-device-plugin` image required GPUs to test, therefore it is not currently covered by automated tests.
+
+This document describes the manual steps taken to test the image on GCP.
+
+Note, for testing we use a the `nvidia/gpu-operator` helm chart which requires bash so you will want to use the `latest-dev` variant.
```
Comment (k4leung4) [inline images/nvidia-device-plugin/tests/README.md:7]: another option is to override the daemonset to call `nvidia-device-plugin` directly

i tested it and the `/bin/entrypoint.sh` provides extra checks but isnt needed really.

File: images/nvidia-device-plugin/tests/README.md (hunk 1 above)
Comment (rawlingsj) [reply images/nvidia-device-plugin/tests/README.md]: god spot! thanks

File: images/nvidia-device-plugin/tests/README.md (hunk 2 above)
Comment (rawlingsj) [reply images/nvidia-device-plugin/tests/README.md:7]: Yeah - I was thinking for now to keep the steps as straight forward as possible as there's a lot going on there.  If you want I can add this in but using `latest-dev` seems like a fair option, also for testing folks might like bash to exec and noodle around the running pod.

Comment (k4leung4) [reply images/nvidia-device-plugin/tests/README.md:7]: sgtm

**Review by k4leung4 — APPROVED** [review]

File: images/nvidia-device-plugin/tests/README.md (hunk 2 above)
Comment (rawlingsj) [reply images/nvidia-device-plugin/tests/README.md:7]: cool thanks 👍

**Pull request merged**